lib/class-wp-rest-plugins-controller.php

```diff
@@ -61,11 +72,63 @@ public function delete_item( $request ) {
 	}
 
 	public function prepare_item_for_response( $item, $request ) {
-
+		return $item;
 	}
 
 	public function get_item_schema() {
-
+		$schema = array(
+			'$schema'              => 'http://json-schema.org/draft-04/schema#',
+			'title'                => 'plugin',
+			'type'                 => 'object',
+			'properties'           => array(
+				'Name'        => array(
```

Comment:
Ah, I misunderstood what you were saying.
Anyway, from https://core.trac.wordpress.org/browser/tags/4.4.1/src//wp-admin/includes/plugin.php#L57: 
```PHP
*     @type string $Name        Name of the plugin. Should be unique.
*     @type string $Title       Title of the plugin and link to the plugin's site (if set).
```